ethereum/sidecar/cli/flags.go

```diff
@@ -12,6 +12,10 @@ const (
 	FlagServerNetwork             = "ethereum-sidecar.server.network"
 	FlagServerBatchSize           = "ethereum-sidecar.server.batch-size"
 	FlagServerRequestsPerMinute   = "ethereum-sidecar.server.requests-per-minute"
+	FlagKeyringBackend            = "keyring-backend"
+	FlagKeyringDir                = "keyring-dir"
+	FlagKeyType                   = "keyring-type"
```

Comment:
We support only ECDSA keys in the sidecar so there is probably no point of exposing this flag. Let's remove it and make sure we fail if non-ECDSA key is loaded upon sidecar startup.